pkg/agent/flowexporter/connections/connections.go

```diff
@@ -190,7 +192,7 @@ func (cs *ConnectionStore) Poll() (int, error) {
 	// We do not expect any error as resetConn is not returning any error
 	cs.ForAllConnectionsDo(resetConn)
 
-	filteredConns, err := cs.connDumper.DumpFlows(openflow.CtZone)
+	filteredConns, conntrackOccupancyMetrics, err := cs.connDumper.DumpFlows(openflow.CtZone)
```

Comment:
same here.. a shorter variable name is better.

Reply:
Thanks, done.